frontend/jest.config.ts

```diff
@@ -138,8 +138,7 @@ const config: Config = {
   // setupFiles: [],
 
   // A list of paths to modules that run some code to configure or set up the testing framework before each test
-  // setupFilesAfterEnv: [],
-
+  // setupFilesAfterEnv: ["./jest.setup.ts"],
```

Comment:
where is this file?

Reply:
frontend root

Reply:
incredihire/Survey/frontend/jest.config.ts

Reply:
jest.setup.ts"

Reply:
the file is missing. or remove this line jest.setup.js

Reply:
it's commented out